spec/std/process_spec.cr

```diff
@@ -2,6 +2,15 @@ require "spec"
 require "process"
 require "tempfile"
 
+private GOOD_USER_NAME = "root"
+{% if flag?(:darwin) || flag?(:openbsd) || flag?(:freebsd) %}
+  private GOOD_GROUP_NAME = "wheel"
+{% elsif flag?(:linux) %}
```

Comment:
redundant `elsif` branch

Reply:
I did this intentionally. Specifically to note that Linux has a specified value. If anything I should remove the `else` so it doesn't compile on other platforms if they haven't specified a good value.

Reply:
Maybe it's better to just raise in the else branch.